packages/block-editor/src/components/border-radius-control/stories/index.story.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BorderRadiusControl from '../';
+
+const meta = {
+	title: 'BlockEditor/BorderRadiusControl',
+	component: BorderRadiusControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component: 'Control to display border radius options.',
+			},
+		},
+	},
+	argTypes: {
+		values: {
+			control: 'object',
+			description: 'Border radius values.',
+			table: {
+				type: { summary: 'object' },
+			},
+		},
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+			table: {
+				type: { summary: 'function' },
+			},
+			description: 'Callback to handle onChange.',
+		},
+	},
+};
+
+export default meta;
+
+export const Default = {
+	render: function Template( { onChange, ...args } ) {
+		const [ values, setValues ] = useState( args.values );
+
+		return (
+			<BorderRadiusControl
+				{ ...args }
+				values={ values }
+				onChange={ ( ...changeArgs ) => {
+					setValues( ...changeArgs );
+					onChange( ...changeArgs );
+				} }
+			/>
+		);
+	},
+	args: {
+		values: {
+			topLeft: '10px',
+			topRight: '10px',
+			bottomLeft: '10px',
+			bottomRight: '10px',
+		},
+	},
+};
```

Comment:
```suggestion
export const Default = {
	render: function Template( { onChange, ...args } ) {
		const [ values, setValues ] = useState();

		return (
			<BorderRadiusControl
				{ ...args }
				values={ values }
				onChange={ ( ...changeArgs ) => {
					setValues( ...changeArgs );
					onChange( ...changeArgs );
				} }
			/>
		);
	},
};
```

As with other stories, I don't think we need to add default values ​​unless we have a specific reason.